src/cache.ts

```diff
@@ -25,6 +25,7 @@ export function getCacheIdentity(entry: EntryConfig): string {
     packageVersion,
   });
 
+  console.log('val', value);
```

Comment:
@smeijer, was this `console.log` left here intentionally? It's currently (as of [v1.29.1](https://www.npmjs.com/package/unimported/v/1.29.1)) present in the published package on npm (see https://www.npmjs.com/package/unimported/v/1.29.1?activeTab=code, `dist/ -> cache.js`, line 27) and may be logging something like the following for users (from [this CI run](https://github.com/ericcornelissen/shescape/actions/runs/5053707632/jobs/9067816836#step:8:10)):

```
- initializing
val {"file":"index.js","aliases":{},"extensions":[".js",".jsx",".ts",".tsx"],"filepath":"/home/runner/work/shescape/shescape/index.js","packageVersion":"1.29.1"}
val {"file":"testing.js","aliases":{},"extensions":[".js",".jsx",".ts",".tsx"],"filepath":"/home/runner/work/shescape/shescape/testing.js","packageVersion":"1.29.1"}
✓ There don't seem to be any unimported files.
```

To me, this doesn't look like something that's logged intentionally.

Reply:
Ooops, no, I wasn't. 😬 Thanks for the report.

Reply:
Fixed with #158. Thanks again.